lib/gitsh/tab_completion/README.md

```diff
@@ -102,6 +102,21 @@ then hit <kbd>tab</kbd>. In this case, we'd pass `add` to the automaton and move
 from state `0` to state `4`. From state `4` the next thing we expect is a path,
 so we'd offer file paths from the current directory as completions.
 
+### Building the state graph
+
+The state graph is built by the `Gitsh::TabCompletion::AutomatonFactory` class.
+The initial implementation is small and fixed, but is expected to grow in size
+and complexity as we support more subtle forms of tab completion.
+
+To help debug the state graph, the repository includes a visualisation tool.
+Running `bin/tcviz` will load the configuration files, and output the graph in
```

Comment:
Good point. Also, it doesn't load any configuration files (yet—that's on the larger branch I extracted this smaller piece from).